docs/sources/alerting/unified-alerting/alerting-rules/create-grafana-managed-rule.md

```diff
@@ -112,6 +112,24 @@ The following template variables are available when expanding annotations and la
 | $values | The values of all reduce and math expressions that were evaluated for this alert rule. For example, `{{ $values.A }}`, `{{ $values.A.Labels }}` and `{{ $values.A.Value }}` where `A` is the `refID` of the expression. This is unavailable when the rule uses a classic condition. |
 | $value  | The value string of the alert instance. For example, `[ var='A' labels={instance=foo} value=10 ]`.                                                                                                                                                                                  |
 
+#### Template functions
```

Comment:
How do you feel about including a list of _all_ the function and explicitly saying which ones we don't support? You can add a new column that reads - "supported" and includes the version in which they were introduced or "No" if we don't support them.

Reply:
What about explicitly saying "Not supported" in the "Description" field?

Reply:
That works for me.